### PR TITLE
Modefy to get all content and change to V2

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/cqwu/news/notify
/cqwu/news/academiceve
/cqwu/news
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [ ] New Route
- [X] Documentation
  - [X] CN
  - [ ] EN
- [X] 全文获取 fulltext
  - [X] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/cqwu/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    notify: '/channel_7721.html',
+    academiceve: '/channel_7722.html',
+};
+const titleMap = {
+    notify: '通知',
+    academiceve: '学术活动',
+};
+
+module.exports = async (ctx) => {
+    const type = ctx.params.type || 'academiceve';
```
Comment (TonyRL) [inline lib/v2/cqwu/index.js:15]:
```suggestion
    const type = ctx.params.type ?? 'academiceve';
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/cqwu/index.js (hunk 1 above)
Comment (TonyRL) [inline lib/v2/cqwu/index.js:15]:
```suggestion
    const type = ctx.params.type ?? 'academiceve';
```

File: lib/v2/cqwu/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/cqwu/news/:type?': ['Fatpandac'],
```
Comment (TonyRL) [inline lib/v2/cqwu/maintainer.js:2]:
```suggestion
    '/news/:type?': ['Fatpandac'],
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/cqwu/index.js (hunk 1 above)
Comment (Fatpandac) [reply lib/v2/cqwu/index.js:15]: Thanks

File: lib/v2/cqwu/maintainer.js (hunk 2 above)
Comment (Fatpandac) [reply lib/v2/cqwu/maintainer.js:2]: Thank you

File: lib/v2/cqwu/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                const page = await got.get(pageUrl);
+                const $ = cheerio.load(page.data);
+                return $('.news-info').html();
+            });
+
+            return {
+                title: $(item).find('a').text(),
+                link: pageUrl,
+                description: desc,
+                pubDate: new Date($(item).find('span').text()).toUTCString(),
```
Comment (TonyRL) [inline lib/v2/cqwu/index.js:35]: Please use `parseDate()`

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Pull request merged**